xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1177 @@
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /var/lib/registry
+  maintenance:
+    readonly:
+      enabled: true
+http:
+  addr: 0.0.0.0:5000
+  headers:
+    X-Content-Type-Options: [nosniff]
+  tls:
+    certificate: /etc/docker/registry/certs/domain.crt
+    key: /etc/docker/registry/certs/domain.key
+health:
+  storagedriver:
+    enabled: true
+    interval: 10s
+threshold: 3
+         </screen>
+      <para>
+       For more details on the configuration, refer to:
+       <link xlink:href="https://docs.docker.com/registry/configuration/">Docker
+       Registry: Configuration</link>
+      </para>
+     </step>
+     <step>
+      <para>
+       Start the registry container.
+      </para>
+<screen>&prompt.user;<command>cd /etc/docker/registry</command>
+&prompt.user;<command>docker run -d -p 5000:5000 --restart=always --name registry \
+-v /etc/docker/registry:/etc/docker/registry:ro \
+-v <replaceable>/var/lib/registry</replaceable>:/var/lib/registry registry.suse.com/sles12/registry:2.6.2</command>
+       </screen>
+     </step>
+    </procedure>
+
+    <para>
+     Now, you should have the registries set up and listening on port
+     <literal>5000</literal> on their respective servers.
+    </para>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.container-registry.client">
+    <title>Client Configuration</title>
+    <para>
+     Using the airgapped mirror works exactly like using a traditional
+     on-premise mirror. You must configure the internal registry mirror in
+     &dashboard;.
+    </para>
+    <procedure>
+     <title>Configuring Registry Mirror</title>
+     <step>
+      <para>
+       Log in to &dashboard; on the &productname; admin node.
+      </para>
+     </step>
+     <step>
+      <para>
+       Navigate to <guimenu>Settings &rarr; Mirrors</guimenu> and create a
+       definition for your internal registry mirror as described in
+       <xref linkend="sec.admin.velum.mirror"/>.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.helm-charts">
+   <title>Helm Chart Repository Mirror</title>
+   <important>
+    <para>
+     To make use of the helm charts, you must complete
+     <xref linkend="sec.deploy.scenarios.airgap.container-registry"/>.
+    </para>
+   </important>
+   <para>
+    The helm charts will require images available from a registry mirror. The
+    charts themselves are served on a simple webserver and do not require any
+    particular configuration apart from basic networking availability and a
+    hostname.
+   </para>
+   <sect3>
+    <title>Mirror Configuration</title>
+    <procedure>
+     <title>Setting Up Chart Repository Mirror</title>
+     <step>
+      <para>
+       We will re-use the webserver that is shipped with &rmtool; to create
+       a virtual host for the chart repository. The repository will be available
+       at <literal>http://mymirror.local:80/charts</literal>.
```

Comment:
This might have to be extended if we prove they are available also over HTTPS (see previous comment).